external-providers/golang-external-provider/pkg/golang/provider.go

```diff
@@ -51,9 +51,14 @@ func (p *golangProvider) Init(ctx context.Context, log logr.Logger, c provider.I
 		os.Setenv(k, v)
 	}
 
+	lspServerPath, ok := c.ProviderSpecificConfig[provider.LspServerPathConfigKey].(string)
+	if !ok || lspServerPath == "" {
+		return nil, fmt.Errorf("invalid lspServerPath provided, unable to init go provider")
```

Comment:
just curious, does this error bubble to the user nicely?

Reply:
@fabianvf yeah, it looks like this: 
```
time="2023-07-17T14:50:07-04:00" level=error msg="unable to load rule set" error="stat /tmp/ruleset.yaml: no such file or directory" logger=parser
time="2023-07-17T14:50:07-04:00" level=error msg="unable to init the providers" error="invalid lspServerPath provided, unable to init java provider" provider=java
```